packages/mui-joy/src/DrawerContent/DrawerContent.tsx

```diff
@@ -0,0 +1,228 @@
+'use client';
```

Comment:
Maybe the DrawerContent is not needed as well.

Reply:
Agreed, removed.